cmd/tools/studio/cfg/copyenv.go

```diff
@@ -0,0 +1,85 @@
+package cfg
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var heyCMD = &cobra.Command{
```

Comment:
```suggestion
var copyenvCMD = &cobra.Command{
```

I assume heyCMD wasn't planned on being the final name :D

Reply:
Lol :) no. Renamed